Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
```

Comment:
Discussed with Zev – the only way I know to do this is to mark these functions as `NS_SWIFT_UNAVAILABLE` and then provide swift code to redefine the functions with default parameters.

I'm hesitant to do this as for people who are using the library in a swift-free app, their app will now need to bundle the swift standard library. Could potentially break this into a swift specific subspec, but it feels pretty messy.

Zev is going to investigate if there are any other ways to tackle this.